Makefile

```diff
@@ -49,7 +49,7 @@ start-s3rver:
 
 test: test-js test-python
 test-js: start-support
-	@yarn turbo run test
+	@yarn turbo run --log-order=stream test
```

Comment:
```
@prairielearn/prairielearn:test: ::error file=/home/runner/work/PrairieLearn/PrairieLearn/apps/prairielearn/src/tests/helperExam.ts,title=src/tests/clientFiles.test.ts > Client files endpoints > startExam-6. POST to assessment URL > should create one assessment_instance,line=171,column=14::AssertionError: expected +0 to equal 1%0A%0A- Expected%0A+ Received%0A%0A- 1%0A+ 0%0A%0A ❯ src/tests/helperExam.ts:171:14%0A%0A
```

This is causing the annotations to not get picked up.

Reply:
Ah, that's unfortunate. Is that because the line has to start with the magic string for it to be picked up?

Is this an argument to spend more time on Vitest projects/workspaces? I really like being able to follow along with output as the job runs, especially in the case of weird timeout/shutdown issues where the process might never exit.

Reply:
I think so.

Reply:
See #12042